loader_hub/whatsapp/base.py

```diff
@@ -0,0 +1,57 @@
+"""Whatsapp chat data loader"""
+
+from pathlib import Path
+from typing import List
+
+from gpt_index.readers.base import BaseReader
+from gpt_index.readers.schema.base import Document
+
+
+class WhatsappChatLoader(BaseReader):
+    """
+    Whatsapp chat data loader.
+
+    Args:
+        path (str): Path to Whatsapp chat file.
+    """
+
+    def __init__(self, path: str):
+        """Initialize with path."""
+        
+        self.file_path = path
+
+    def load_data(self, verbose=False) -> List[Document]:
```

Comment:
Using `logging` also removes the need to have a `verbose` argument here, so we can take that out.